### PR TITLE
Added the delete message function

### PR DESCRIPTION
**Pull request by kvyatkovskys**
Added a function for deleting messages from the chat.

### CODE REVIEW

File: src/morse/api.clj [hunk 1]
```diff
@@ -56,6 +56,15 @@
           ]
          (-> resp :body))))
 
+(defn delete-text
+  "Removing a message from the chat"
+  [token chat-id message-id]
+  (let [base-url "https://api.telegram.org/bot"
```
Comment (Otann) [inline src/morse/api.clj]: I believe there is an existing `base-url` at the top of the file ;)

File: src/morse/api.clj [hunk 2]
```diff
@@ -56,6 +56,15 @@
...
+        url (str base-url token "/deleteMessage")
+        query (into {:chat_id chat-id :message_id message-id} {})
```
Comment (Otann) [inline src/morse/api.clj]: Why are you inserting an empty map into a map? I think it's unnecessary

**Comment by Otann** [comment]
Hi! thank you for your contribution! I was not aware that deleting is now available through API.
I asked for some cosmetic changes for the code. Could you please also add a basic test for this?

**Comment by kvyatkovskys** [comment]
Hi! I added the test and removed some excess code :)

**Comment by Otann** [comment]
@kvyatkovskys thank you man!
Released `0.2.8` to clojars with this

**Pull request merged**